Update id.m3u

F640 opened the pull request:
Changes:
- Added local channels (most of it are not 24/7)
- Added some satellite and cable channels
- Edited some national channels
- Removed DRM'ed streams (mostly `liveanevia.mncnow.id`)

Notes:
- [MAXStream](https://maxstream.tv) (`cdn-telkomsel-01.akamaized.net`), [YouTV](https://youtv.live/) (`h1.intechmedia.net`) and Rodja (`tv.rodja.live`) links are geo-blocked.
- When a channel on [JPM Stream](https://jpmstream.com/) (`122.248.43.138`) sign off it gives error 404, but sometimes it's offline even it's not signed off yet. Some other streams in this PR also act like that.
- Hope i didn't miss anything here

This PR related to #4794

F640 committed:
Rough (quick) additions

Added channels from my scratch note.

F640 committed:
Clean-up playlist

F640 committed:
Edited id.m3u

F640 committed:
Reordered and deleted some channels

F640 committed:
Added Dangdut Tivi and Lombok TV

F640 commented:
> > * AFB TV (playable by streamtest.in, but not VLC): https://5e396e0211d7a.streamlock.net/afbtv/live/playlist.m3u8 and https://wowza58.indostreamserver.com/afbtv/live/playlist.m3u8
> 
> ![image](https://user-images.githubusercontent.com/74099190/166933337-7b147946-5ffc-43d9-9d59-a6a79d604156.png) SSL was expired, so this link is not playable

I know that, but you can ignore it for 24 hours on Android version of VLC

F640 committed:
Added AFB TV

LaneSh4d0w commented on streams/id.m3u:
```diff
@@ -13,60 +57,235 @@ http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
+#EXTINF:-1 tvg-id="DaAiTVIndonesia.id" status="error",DAAI TV Indonesia (720p) [Not 24/7]
+https://dacastmmd.mmdlive.lldns.net/dacastmmd/bf3a76722204472aa45938f2b67ef808/playlist.m3u8
+#EXTINF:-1 tvg-id="DangdutTivi.id" status="error",Dangdut Tivi (360p)
+https://bravo.kugo.id/dangduttivi/udp.stream_360p/chunklist.m3u8
```
```suggestion
https://bravo.kugo.id/dangduttivi/udp.stream/playlist.m3u8
```

LaneSh4d0w commented on streams/id.m3u:
```diff
@@ -13,60 +57,235 @@ http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
+http://103.255.15.222:1935/tv/jitv_720p/playlist.m3u8
+#EXTINF:-1 tvg-id="JogjaIstimewaTV.id" status="error",Jogja Istimewa TV (360p) [Geo-blocked]
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (X11; Linux x86_64)
+https://h1.intechmedia.net/intech/ch65.m3u8
```
CH78 has something... however can't seem to see anything.

F640 replied:
https://h1.intechmedia.net/intech/ch78.m3u8 = TVRI

F640 replied:
Oh wait, that's TVRI East Java. I was wrong. Sorry

F640 replied:
It's very easy to find the link for these streams anyways, visit https://youtv.live/, then view source of the page. Remember to use browser user agent and no referrer or referrer set to"https://youtv.live/" while playing it on VLC. Otherwise you will be redirected to YouTV app installation procedure (especially links that starts with "https://h1.intechmedia.net/nasional/").

F640 replied:
Assuming the is no problem in this review anymore

LaneSh4d0w commented:
> > * AFB TV (playable by streamtest.in, but not VLC): https://5e396e0211d7a.streamlock.net/afbtv/live/playlist.m3u8 and https://wowza58.indostreamserver.com/afbtv/live/playlist.m3u8
> 
> ![image](https://user-images.githubusercontent.com/74099190/166933337-7b147946-5ffc-43d9-9d59-a6a79d604156.png) SSL was expired, so this link is not playable

http://5e396e0211d7a.streamlock.net:1935/afbtv/live/playlist.m3u8 most certainly.

LaneSh4d0w commented on streams/id.m3u:
```diff
@@ -1,6 +1,50 @@
+#EXTINF:-1 tvg-id="BantenTV.id" status="error",Banten TV (720p) [Not 24/7]
+https://v2.siar.us/barayatv/live/chunks.m3u8
+#EXTINF:-1 tvg-id="BatamTV.id" status="error",Batam TV (720p) [Not 24/7]
+http://122.248.43.138:1935/ch5/myStream/playlist.m3u8
```
http://122.248.43.138:1935/ch4/myStream/playlist.m3u8 Sempang TV

http://122.248.43.138:1935/ch2/myStream/playlist.m3u8 jTV

LaneSh4d0w commented on streams/id.m3u:
```diff
@@ -13,60 +57,235 @@ http://edge.linknetott.swiftserve.com/live/BsNew/amlst:bsenglish/playlist.m3u8
+#EXTINF:-1 tvg-id="IzzahTV.id" status="error",Izzah TV (480p)
+https://streaming.radiosalamjambi.com/izzahtv.m3u8
+#EXTINF:-1 tvg-id="JTV.id" status="error",JTV (1080p) [Not 24/7]
+https://cdn02.mola.tv/Content/HLS/Live/channel(f2cd666d-f350-1140-7ce5-3d7f39f84727)/08.m3u8
```
```suggestion
https://cdn02.mola.tv/Content/HLS/Live/channel(f2cd666d-f350-1140-7ce5-3d7f39f84727)/index.m3u8
```

F640 committed:
Edited a channel (PR suggestion)

Co-authored-by: LaneSh4d0w <lanesh4d0w@gmail.com>

F640 committed:
Edited a channel (PR suggestion from maintainer)

Co-authored-by: LaneSh4d0w <lanesh4d0w@gmail.com>

F640 committed:
Edited some channels url and removed dulicates

F640 commented:
> > > * AFB TV (playable by streamtest.in, but not VLC): https://5e396e0211d7a.streamlock.net/afbtv/live/playlist.m3u8 and https://wowza58.indostreamserver.com/afbtv/live/playlist.m3u8
> > 
> > 
> > ![image](https://user-images.githubusercontent.com/74099190/166933337-7b147946-5ffc-43d9-9d59-a6a79d604156.png) SSL was expired, so this link is not playable
> 
> http://5e396e0211d7a.streamlock.net:1935/afbtv/live/playlist.m3u8 most certainly.

Too late, i've added that: F640@71232f9bbfcea773b546370fd2c8e7d50ab6667c

F640 committed:
Added Ainos TV

F640 committed:
Added Duta TV

Finally it's online in signed off state (tested 
right before this commit).

LaneSh4d0w commented:
I've scanned the intechmedia channels, found around 40 channels. Going to add them after verification.

F640 commented:
> I've scanned the intechmedia channels, found around 40 channels. Going to add them after verification.
> 
> 1 - Insan TV 2 - Ashiil TV 3 - Surau TV 4 - Ahsan TV 5 - Tadabbur TV 6 - Rasyaad TV 7 - Yufid TV 8 - Hijrah TV 9 - Al-Iman TV | does not answer | 10 12 13 14 16 17 18 19 20 21 22 23 | answers | 24 - Al-Furqon TV 25 - MGI TV | does not answer (again) | 26 27 28 29 30 51 52 58 59 | answers | 61 - TVRI Nasional 62 - TVRI Sumatera Selatan 64 - Al-Rahjah TV 65 - jitv 67 - TVRI World 74 - TVRI Jawa Babay 77 - TVRI 78 - TVRI 80 - ATV 82 - TVRI 86 - TVRI 88 - TVRI World

Answers are here below:
>  | does not answer |
> 10 12 13 14 16 17 18 19 20 21 22 23

10 - Niaga TV
12 - Salam TV
13 - Madinah TV
14 - Mekkah TV (Al Quran Al Kareem)
16 - RadioKita TV
17 - Sunnah Jalan Pasti TV
18 - Wesal TV
19 - MHO TV
20 - Bina Madani TV
22 - Bin Baz TV
23 - Media Sunnah Aceh TV

> | does not answer (again) |
> 26 27 28 29 30 51 52 58 59

26 - Rodja TV
27 - UFA 24
28 - jitv
29 - Jaya TV Jayapura (or Jaya TV)
30 - BETV (Bengkulu Ekspres TV)
51 - DM TV Malang (or DM TV)
52 - Jaya TV Jayapura (or Jaya TV)
58 - BETV (Bengkulu Ekspres TV)
59 - RBTV (Rakyat Bengkulu TV)

> 74 - TVRI Jawa Babay

Did you mean TVRI Jawa barat (TVRI West Java)?

> 77 - TVRI
> 78 - TVRI

> 82 - TVRI
> 86 - TVRI

77 - TVRI Jawa Tengah (TVRI Central Java)
78 - TVRI Jawa Timur (TVRI East Java)
82 - TVRI Papua Barat (TVRI West Papua)
86 - TVRI Sulawesi Selatan (TVRI South Sulawesi)

> 88 - TVRI World

TVRI Bali. This channel shares time between three channels in one channel which are: TVRI Nasional, TVRI Lokal (in this case Bali), and TVRI World.

You might want to scan this link below as well:
https://h1.intechmedia.net/nasional/ch3.m3u8
Remember to use browser user agent and no referrer or referrer set to "https://youtv.live/" while playing it on VLC. Otherwise you will be redirected to YouTV app installation procedure video

F640 committed:
Added Dayak TV

This one online right before i made this commit

LaneSh4d0w commented:
> > I've scanned the intechmedia channels, found around 40 channels. Going to add them after verification.
> > 1 - Insan TV 2 - Ashiil TV 3 - Surau TV 4 - Ahsan TV 5 - Tadabbur TV 6 - Rasyaad TV 7 - Yufid TV 8 - Hijrah TV 9 - Al-Iman TV | does not answer | 10 12 13 14 16 17 18 19 20 21 22 23 | answers | 24 - Al-Furqon TV 25 - MGI TV | does not answer (again) | 26 27 28 29 30 51 52 58 59 | answers | 61 - TVRI Nasional 62 - TVRI Sumatera Selatan 64 - Al-Rahjah TV 65 - jitv 67 - TVRI World 74 - TVRI Jawa Babay 77 - TVRI 78 - TVRI 80 - ATV 82 - TVRI 86 - TVRI 88 - TVRI World
> 
> Answers are here below:
> 
> > | does not answer |
> > 10 12 13 14 16 17 18 19 20 21 22 23
> 
> 10 - Niaga TV 12 - Salam TV 13 - Madinah TV 14 - Mekkah TV (Al Quran Al Kareem) 16 - RadioKita TV 17 - Sunnah Jalan Pasti TV 18 - Wesal TV 19 - MHO TV 20 - Bina Madani TV 22 - Bin Baz TV 23 - Media Sunnah Aceh TV
> 
> > | does not answer (again) |
> > 26 27 28 29 30 51 52 58 59
> 
> 26 - Rodja TV 27 - UFA 24 28 - jitv 29 - Jaya TV Jayapura (or Jaya TV) 30 - BETV (Bengkulu Ekspres TV) 51 - DM TV Malang (or DM TV) 52 - Jaya TV Jayapura (or Jaya TV) 58 - BETV (Bengkulu Ekspres TV) 59 - RBTV (Rakyat Bengkulu TV)
> 
> > 74 - TVRI Jawa Babay
> 
> Did you mean TVRI Jawa barat (TVRI West Java)?
> 
> > 77 - TVRI
> > 78 - TVRI
> 
> > 82 - TVRI
> > 86 - TVRI
> 
> 77 - TVRI Jawa Tengah (TVRI Central Java) 78 - TVRI Jawa Timur (TVRI East Java) 82 - TVRI Papua Barat (TVRI West Papua) 86 - TVRI Sulawesi Selatan (TVRI South Sulawesi)
> 
> > 88 - TVRI World
> 
> 88 - TVRI Bali. This channel shares time between three channels in one channel which are: TVRI Nasional, TVRI lokal (in this case TVRI Bali), and TVRI World.
> 
> You might want to scan this link below as well: https://h1.intechmedia.net/nasional/ch3.m3u8 Remember to use browser user agent and no referrer or referrer set to "https://youtv.live/" while playing it on VLC. Otherwise you will be redirected to YouTV app installation procedure video

Not easy with the resolution to read properly to be honest. Most certainly is what you meant.
Going to look after the other ones soon.

F640 commented:
https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(0b50f52d-2491-4050-81dd-79045ddcfb34)/index.m3u8 (Hi Indo!)
Not sure will add this since i remember this channel was in MAXStream. It has been gone now, but the stream link still alive.

F640 commented:
> Not easy with the resolution to read properly to be honest.

I understand that, it's not really easy to identify logos and text in fairly low resolution videos and pictures.

> https://cdn-telkomsel-01.akamaized.net/Content/HLS/Live/channel(0b50f52d-2491-4050-81dd-79045ddcfb34)/index.m3u8 (Hi Indo!) Not sure will add this since i remember this channel was in MAXStream. It has been gone now, but the stream link still alive.

F640 committed:
Added some channels

F640 committed:
Added Zee Bioskop

F640 committed:
Added One TV

F640 committed:
Added Dens TV in-house channels.

F640 commented:
https://h1.intechmedia.net/intech/kemenparekraf.m3u8 (Kemenparekaf)

Not sure about adding this too.

F640 committed:
Added Bali TV not geo-blocked stream

MNCNow still has unprotected stream, maybe the last one? That's good, hope it won't die or suddenly protected with DRM.

LaneSh4d0w commented:
there's one at https://liveanevia.mncnow.id/live/eds/BaliTV/sa_hls_vmx/BaliTV.m3u8 but it doesn't work.

BellezaEmporium committed:
Merge branch 'master' into F640-patch-1

LaneSh4d0w commented:
> > I've scanned the intechmedia channels, found around 40 channels. Going to add them after verification.
> > 1 - Insan TV 2 - Ashiil TV 3 - Surau TV 4 - Ahsan TV 5 - Tadabbur TV 6 - Rasyaad TV 7 - Yufid TV 8 - Hijrah TV 9 - Al-Iman TV | does not answer | 10 12 13 14 16 17 18 19 20 21 22 23 | answers | 24 - Al-Furqon TV 25 - MGI TV | does not answer (again) | 26 27 28 29 30 51 52 58 59 | answers | 61 - TVRI Nasional 62 - TVRI Sumatera Selatan 64 - Al-Rahjah TV 65 - jitv 67 - TVRI World 74 - TVRI Jawa Babay 77 - TVRI 78 - TVRI 80 - ATV 82 - TVRI 86 - TVRI 88 - TVRI World
> 
> Answers are here below:
> 
> > | does not answer |
> > 10 12 13 14 16 17 18 19 20 21 22 23
> 
> 10 - Niaga TV 12 - Salam TV 13 - Madinah TV 14 - Mekkah TV (Al Quran Al Kareem) 16 - RadioKita TV 17 - Sunnah Jalan Pasti TV 18 - Wesal TV 19 - MHO TV 20 - Bina Madani TV 22 - Bin Baz TV 23 - Media Sunnah Aceh TV
> 
> > | does not answer (again) |
> > 26 27 28 29 30 51 52 58 59
> 
> 26 - Rodja TV 27 - UFA 24 28 - jitv 29 - Jaya TV Jayapura (or Jaya TV) 30 - BETV (Bengkulu Ekspres TV) 51 - DM TV Malang (or DM TV) 52 - Jaya TV Jayapura (or Jaya TV) 58 - BETV (Bengkulu Ekspres TV) 59 - RBTV (Rakyat Bengkulu TV)
> 
> > 74 - TVRI Jawa Babay
> 
> Did you mean TVRI Jawa barat (TVRI West Java)?
> 
> > 77 - TVRI
> > 78 - TVRI
> 
> > 82 - TVRI
> > 86 - TVRI
> 
> 77 - TVRI Jawa Tengah (TVRI Central Java) 78 - TVRI Jawa Timur (TVRI East Java) 82 - TVRI Papua Barat (TVRI West Papua) 86 - TVRI Sulawesi Selatan (TVRI South Sulawesi)
> 
> > 88 - TVRI World
> 
> 88 - TVRI Bali. This channel shares time between three channels in one channel which are: TVRI Nasional, TVRI lokal (in this case TVRI Bali), and TVRI World.
> 
> You might want to scan this link below as well: https://h1.intechmedia.net/nasional/ch3.m3u8 Remember to use browser user agent and no referrer or referrer set to "https://youtv.live/" while playing it on VLC. Otherwise you will be redirected to YouTV app installation procedure video

Going to look at your link.

F640 committed:
Added, edited and rearranged some channels

F640 committed:
Added TV Edukasi

F640 committed:
Added, edited, and rearranged some channels

LaneSh4d0w commented:
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",PAL TV PALEMBANG
http://122.248.43.138:1935/ch7/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",JEK TV JAMBI
http://122.248.43.138:1935/ch14/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",JAMBI TV
http://122.248.43.138:1935/ch23/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",BETV Bengkulu
http://122.248.43.138:1935/ch3/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RBTV Bengkulu
http://122.248.43.138:1935/ch12/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RADAR LAMPUNG
http://122.248.43.138:1935/ch17/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RT6V RIAU
http://122.248.43.138:1935/ch11/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",SMTV SUMEDANG
http://122.248.43.138:1935/ch25/myStream/live.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",KAWANUA TV
http://122.248.43.138:1935/ch18/myStream/playlist.m3u8
#EXTINF:-1 tvg-logo="https://i.ibb.co/y685Xr1/JTV.png" group-title="LocalDaerah", JTV
http://122.248.43.138:1935/ch2/myStream/playlist.m3u8
#EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",BTV BALIK PAPAN
http://122.248.43.138:1935/ch10/myStream/playlist.m3u8

F640 commented:
> #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",PAL TV PALEMBANG http://122.248.43.138:1935/ch7/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",JEK TV JAMBI http://122.248.43.138:1935/ch14/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",JAMBI TV http://122.248.43.138:1935/ch23/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",BETV Bengkulu http://122.248.43.138:1935/ch3/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RBTV Bengkulu http://122.248.43.138:1935/ch12/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RADAR LAMPUNG http://122.248.43.138:1935/ch17/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RT6V RIAU http://122.248.43.138:1935/ch11/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",SMTV SUMEDANG http://122.248.43.138:1935/ch25/myStream/live.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",KAWANUA TV http://122.248.43.138:1935/ch18/myStream/playlist.m3u8 #EXTINF:-1 tvg-logo="https://i.ibb.co/y685Xr1/JTV.png" group-title="LocalDaerah", JTV http://122.248.43.138:1935/ch2/myStream/playlist.m3u8 #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",BTV BALIK PAPAN http://122.248.43.138:1935/ch10/myStream/playlist.m3u8
> 
> from https://glot.io/snippets/fm4d6ro815
 
Those links already in my commit, they are jpm stream links. Some channels in this links goes offline randomly for unknown reason.

> #EXTINF:-1 tvg-logo="https://imgur.com/wPRYoJE" group-title="LocalDaerah",RT6V RIAU
http://122.248.43.138:1935/ch11/myStream/live.m3u8

Riau TV

F640 committed:
Added another Riau TV URL

For alternative if main (jpm stream) link offline

LaneSh4d0w commented:
> https://youtv.live/

1 to 12.

F640 commented:
ANTV
http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=72da7fd6d1302c0a159f6436d01e9eb0446503011668def857b1c03bafabba51

SCTV
http://akudandia.xyz/locs/akudandia-204.m3u8
http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=274ad4786c3abca69fa097b85867d9a4446503011668def857b1c03bafabba51

Indosiar
http://akudandia.xyz/locs/akudandia-205.m3u8
http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=eae27d77ca20db309e056e3d2dcd7d69446503011668def857b1c03bafabba51

O Channel
http://akudandia.xyz/locs/akudandia-206.m3u8

O Shop
http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=4249a84bdaf63c34332d1988244fb089446503011668def857b1c03bafabba51

Mentari TV
http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=7dd2ae7db7d18ee7c9425e38df1af5e2446503011668def857b1c03bafabba51

TVOne
http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=6e0721b2c6977135b916ef286bcb49ec446503011668def857b1c03bafabba51

From https://pastebin.com/4N2C8h2Y, http://hi-back.masuk.web.id/CODAY.m3u8 and https://yamcode.com/pcb-25979

Likely not a long term URLs since they're ~illegal~ scrapers
Those use  Vidio's stream. Their channels (SCTV, Indosiar, O Channel, and Mentari TV) are geo-blocked.

F640 commented:
> > https://youtv.live/
> 
> 1 to 12.
> 
> #EXTINF:-1, 1 https://h1.intechmedia.net/nasional/ch1.m3u8 #EXTINF:-1, 2 https://h1.intechmedia.net/nasional/ch2.m3u8 #EXTINF:-1, 3 https://h1.intechmedia.net/nasional/ch3.m3u8 #EXTINF:-1, 4 https://h1.intechmedia.net/nasional/ch4.m3u8 #EXTINF:-1, 5 https://h1.intechmedia.net/nasional/ch5.m3u8 #EXTINF:-1, 6 https://h1.intechmedia.net/nasional/ch6.m3u8 #EXTINF:-1, 7 https://h1.intechmedia.net/nasional/ch7.m3u8 #EXTINF:-1, 8 https://h1.intechmedia.net/nasional/ch8.m3u8 #EXTINF:-1, 9 https://h1.intechmedia.net/nasional/ch9.m3u8 #EXTINF:-1, 10 https://h1.intechmedia.net/nasional/ch10.m3u8 #EXTINF:-1, 11 https://h1.intechmedia.net/nasional/ch11.m3u8 #EXTINF:-1, 12 https://h1.intechmedia.net/nasional/ch12.m3u8

1 - Kompas TV
2 - SCTV
3 - Indosiar
4 - O Channel
5 - Trans TV
6 - Trans 7
7 - ANTV
8 - tvOne
9 - Metro TV
10 - CNN Indonesia
11 - NET.
12 - RTV (Rajawali TV)

LaneSh4d0w commented:
> ANTV http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=72da7fd6d1302c0a159f6436d01e9eb0446503011668def857b1c03bafabba51
> 
> SCTV http://akudandia.xyz/locs/akudandia-204.m3u8 http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=274ad4786c3abca69fa097b85867d9a4446503011668def857b1c03bafabba51
> 
> Indosiar http://akudandia.xyz/locs/akudandia-205.m3u8 http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=eae27d77ca20db309e056e3d2dcd7d69446503011668def857b1c03bafabba51
> 
> O Channel http://akudandia.xyz/locs/akudandia-206.m3u8
> 
> O Shop http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=4249a84bdaf63c34332d1988244fb089446503011668def857b1c03bafabba51
> 
> Mentari TV http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=7dd2ae7db7d18ee7c9425e38df1af5e2446503011668def857b1c03bafabba51
> 
> TV MUI http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=ffd67d44870fda644560c47682a87a73&cdy=cb70ab375662576bd1ac5aaf16b3fca4dfdded94c48ea6229c48edefa9ec1c9f
> 
> TVOne http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=6e0721b2c6977135b916ef286bcb49ec446503011668def857b1c03bafabba51
> 
> From https://pastebin.com/4N2C8h2Y, http://hi-back.masuk.web.id/CODAY.m3u8 and https://yamcode.com/pcb-25979
> 
> Likely not a long term URLs since they're ~illegal~ scrapers Those use Vidio or UseeTV stream. Some channels (like SCTV, Indosiar, O Channel, and Mentari TV) are geo-blocked, especially UseeTV ones (like TV MUI).

They're most certainly using the same tech as mine, sort of. Mine works with Vidio, not with USee.

F640 commented:
> > ANTV http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=72da7fd6d1302c0a159f6436d01e9eb0446503011668def857b1c03bafabba51
> > SCTV http://akudandia.xyz/locs/akudandia-204.m3u8 http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=274ad4786c3abca69fa097b85867d9a4446503011668def857b1c03bafabba51
> > Indosiar http://akudandia.xyz/locs/akudandia-205.m3u8 http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=eae27d77ca20db309e056e3d2dcd7d69446503011668def857b1c03bafabba51
> > O Channel http://akudandia.xyz/locs/akudandia-206.m3u8
> > O Shop http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=4249a84bdaf63c34332d1988244fb089446503011668def857b1c03bafabba51
> > Mentari TV http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=7dd2ae7db7d18ee7c9425e38df1af5e2446503011668def857b1c03bafabba51
> > TV MUI http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=ffd67d44870fda644560c47682a87a73&cdy=cb70ab375662576bd1ac5aaf16b3fca4dfdded94c48ea6229c48edefa9ec1c9f
> > TVOne http://hi.hi-back.masuk.web.id/c.m3u8?shinta=2022&coday=f846bc691110108f73cb55c2f4049864&cdy=6e0721b2c6977135b916ef286bcb49ec446503011668def857b1c03bafabba51
> > From https://pastebin.com/4N2C8h2Y, http://hi-back.masuk.web.id/CODAY.m3u8 and https://yamcode.com/pcb-25979
> > Likely not a long term URLs since they're ~illegal~ scrapers Those use Vidio or UseeTV stream. Some channels (like SCTV, Indosiar, O Channel, and Mentari TV) are geo-blocked, especially UseeTV ones (like TV MUI).
> 
> They're most certainly using the same tech as mine, sort of. Mine works with Vidio, not with USee.

I forgot to say that Usee stream hates some user agents like VLC similar to YouTV, while Vidio didn't

F640 committed:
Updated r c t i  + streams

They have changed their streams recently

F640 committed:
Changed some stream quality label

F640 committed:
Update id.m3u

F640 committed:
Updated One TV channel ID

Should this channel move to somewhere?

F640 committed:
Update id.m3u

F640 committed:
Merged Intechmedia (YouTV) channels

Same as https://github.com/iptv-org/iptv/pull/7239/commits/fb0e11abf1a37cc9e492c20e50a8f2d7b96bb7fa

BellezaEmporium committed:
Merge branch 'master' into F640-patch-1

BellezaEmporium committed:
Merge branch 'master' into F640-patch-1
